Gruntfile.js

```diff
@@ -384,7 +393,7 @@ module.exports = function (grunt) {
 		watch: {
 			sass: {
 				files: [templateScss + '/**/*.scss', templateJs + '/js-css/*.scss'],
-				tasks: ['scssGlobber','sass']
+				tasks: ['sassGlobber:all','sass']
```

Comment:
This doesn't appear to be a documentation change. What is the purpose of this change?

Reply:
Just making the task scope be more specific and only run the 'all' task defined under scssGlobber instead of running the entire scssGlobber task (at this time there's only 'all' subtask, future there could be others).